tests/test_parser_steps.py

```diff
@@ -0,0 +1,600 @@
+import unittest
+from tests import common
+
+
+class TestStepsParser(unittest.TestCase):
+    """
+        Test steps gherkin parser functionality of romaine's core.
+    """
+    def setUp(self):
+        """
+            Prepare the environment for testing.
+        """
+        # We're doing a lot of long tests- don't limit the diff output length
+        self.maxDiff = None
+
+    def tearDown(self):
```

Comment:
It could, I just quite like having it there to be explicit and provide a nice framework for new tests.